docs/firefox-accounts.rst

```diff
@@ -0,0 +1,86 @@
+.. This Source Code Form is subject to the terms of the Mozilla Public
+.. License, v. 2.0. If a copy of the MPL was not distributed with this
+.. file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+.. _firefox-accounts:
+
+============================
+Firefox Accounts Signup Form
+============================
+
+Introduction
+------------
+
+Certain bedrock pages such as ``/firstrun`` feature a Firefox Accounts signup
```

Comment:
Only small nit would be qualifying this URL for new contributors - `/firefox/firstrun/`. Or possibly even changing it to `/firefox/accounts/`, as that one is even easier to test.